### PR TITLE
Group 1 - Solution for Language Detector

### PR DESCRIPTION
**Pull request by phillipdwright**
- Phil Wright
- David Granas
- Amir Beheshtaein

We score each language in `detect_language()`, and we return the language that scores the highest.  We decided to return a set of matching languages if two languages score the same.  We added a test case for this to `test_main.py` (`test_detect_language_ambiguous()`).

### CODE REVIEW

File: language_detector/main.py [hunk 1]
```diff
@@ -1,9 +1,61 @@
...
+    
+    # get the set of languages that had the top score
+    top_languages = score_dict[max(score_dict)]
+    
+    # if there is a single language with the best score, return that name as a string
+    if len(top_languages) == 1:
+        return next(iter(top_languages))
+    
+    # in case of a tie with multiple languages, return a set of language names    
+    return top_languages
```
Comment (ghost) [inline language_detector/main.py:46]: This looks like great code, I'm not sure all the cleaning was necessary, but otherwise it is simple

Comment (bdauer) [inline language_detector/main.py:46]: I agree with @monkeyarms88.

**Comment by an00bhav** [comment]
Excellent. Good idea to use .lower to convert all the words in lowercase. ( I missed that :P)
Good work, your code seem to be more cleaner.

**Comment by an00bhav** [comment]
Done Review! :)

**Comment by santiagobasulto** [comment]
Good stuff guys, closing it now.

**Pull request closed without merging**